packages/__tests__/src/3-runtime-html/repeat.duplicates.spec.ts

```diff
@@ -100,5 +100,95 @@ describe("3-runtime-html/repeat.duplicates.spec.ts", function () {
 
       assertText('0-a 1-b 2-c ');
     });
+
+    it('object duplication unshift (issue 2078)', function () {
```

Comment:
nit: normally gh describes its links as `#___`, maybe we should have something similar, so searching will be easier
```suggestion
    it('object duplication unshift (issue #2078)', function () {
```